sql/statement_test.go

```diff
@@ -28,19 +28,19 @@ import (
 )
 
 type cursorBufferSizeTestCase struct {
-	Value     int
-	Target    *int32
 	ErrString string
+	Value     int
+	Target    int32
 }
 
-func TestSQLOptions_SetCursorBufferSize(t *testing.T) {
+func TestStatement_SetCursorBufferSize(t *testing.T) {
```

Comment:
Unit tests are nice. However, I couldn't find anything that statement is processed as should be. A test which verifies the parameters are passed to server would be nice, on invocation level or as an integration.

Reply:
That's a fair point, I think we should invest in the "invocation level tests" as you mentioned.

For now, we only have unit tests for the extraction part of these values.
https://github.com/utku-caglayan/hazelcast-go-client/blob/sql-qpi/internal/sql/driver/driver_test.go#L326